data/mods/Magiclysm/monsters/dragon.json

```diff
@@ -1,4 +1,41 @@
 [
+  {
+    "id": "fd_fright",
+    "type": "field_type",
+    "legacy_enum_id": 12,
```

Comment:
Is this necessary? I thought this was only for moving fields over from older games.

Reply:
copy paste error